lib/api/web-element/commands/getRect.js

```diff
@@ -29,6 +29,7 @@
  * @syntax browser.element(selector).getRect()
  * @see https://www.w3.org/TR/webdriver#dfn-get-element-rect
  * @returns {ScopedValue<{ width: number, height: number }>}
+ * @alias rect
```

Comment:
No need to add them in JSDoc, we are only adding these extra aliases for compatibility with global `element()` API so that it is easier for users to migrate from global `element()` API to the new Element API.

Reply:
Better to add `getSize` and `getLocation` as alias here and omit `rect`.

Reply:
updated in [afcfe90](https://github.com/nightwatchjs/nightwatch/pull/4128/commits/afcfe9006b643700d85a0713c56465f192074489)